components/cam/bld/configure

```diff
@@ -1527,21 +1496,12 @@ else {
             $nadv += 8;
             if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
         }
-        elsif ($microphys_pkg =~/^p3/) {
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
ditto